### DISCUSSION TITLE
Error when implement

### DISCUSSION THREAD

**Discussion by 7maucdn** [tailwindlabs/tailwindcss#6070] [Help]
I have added this plugin to my project as the code below

`const colors = require('tailwindcss/colors')

module.exports = {
    theme: {
        borderColor: theme => ({
            ...theme('colors'),
            'cd': '#0097ff',
        }),
        extend: {
            lineClamp: {
                7: '7',
                8: '8',
                9: '9',
                10: '10',
            }
        }
    },
    plugins: [
        require('@tailwindcss/line-clamp')
    ],
}`

When i run my app It display this error

"Error: tailwindcss plugin function argument object prop "lineClamp" not implemented"

Please help me fix this, thanks in advance.

**Comment by me-nkr** [comment]
was this fixed?
